src/collision/collision.c

```diff
@@ -0,0 +1,48 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   collision.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ycarro <ycarro@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/02/23 14:59:18 by ycarro            #+#    #+#             */
+/*   Updated: 2023/02/23 17:12:59 by ycarro           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <collision.h>
+
+int	it_collides(t_ray *ray_data, char **map, t_point *collision)
```

Comment:
boolean func should have a "is" prefix -> is_collides
and you can use the t_bool object that is defined in macro.h

Reply:
True, solved